src/google/protobuf/stubs/atomicops.h

```diff
@@ -63,28 +63,21 @@ namespace google {
 namespace protobuf {
 namespace internal {
 
-#if defined(GOOGLE_PROTOBUF_ARCH_POWER)
-#if defined(_LP64) || defined(__LP64__)
-typedef int32 Atomic32;
-typedef intptr_t Atomic64;
+#if defined(GOOGLE_PROTOBUF_ARCH_32_BIT) && GOOGLE_PROTOBUF_ARCH_32_BIT
```

Comment:
Simply "#ifdef GOOGLE_PROTOBUF_ARCH_32_BIT"? We don't actually use the macro's value.